fuzz/tls_init/fuzz_init.cc

```diff
@@ -0,0 +1,113 @@
+/* Copyright (c) 2021 Intel Corporation
+ * Copyright (c) 2020-2021 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+extern "C" {
+#include <stdint.h>
+#include <stddef.h>
+#include <string.h>
+#include "rats-tls/api.h"
+#include "rats-tls/log.h"
+#include "rats-tls/claim.h"
+#include "internal/core.h"
+}
+#include <fuzzer/FuzzedDataProvider.h>
+#include <vector>
+
+#define CUSTOM_CLAIMS_SIZE 10
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+	rats_tls_conf_t conf; // consume 192 bytes
+	if (size < sizeof(rats_tls_conf_t) + 10 * sizeof(claim_t) + 50 * 10 + 100) {
+		return 0;
+	}
+	FuzzedDataProvider fuzzed_data(data + sizeof(conf), size - sizeof(conf));
```

Comment:
Why skip `sizeof(conf)` here? This appears to be part of the old code because the skipped bytes are not used later.